target/linux/ipq807x/files/arch/arm64/boot/dts/qcom/ipq8072-ax880.dts

```diff
@@ -0,0 +1,370 @@
+// SPDX-License-Identifier: MIT
```

Comment:
Dual licensing is safer as compatibility isn't based on current licenses interpretation. Do you think you could release it as `GPL-2.0-or-later OR MIT`?

Reply:
no problem.